automation_tools/__init__.py

```diff
@@ -2121,8 +2147,15 @@ def satellite6_upgrade(admin_password=None):
     BASE_URL
         Optional, defaults to available satellite version in CDN.
         URL for the compose repository.
+    FROM_VERSION
+        Satellite current version, to disable repos while upgrading.
+        e.g '6.1','6.0'
 
```

Comment:
@elyezer will not be happy about this empty line, but not a blocker for this pr :)